keysign/discover.py

```diff
@@ -42,13 +42,15 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
-            msg_tuple = yield self.bt.find_key(self.bt_code, self.mac)
-            key_data, success, message = msg_tuple
-            if key_data:
+            if BluetoothReceive is not None:
```

Comment:
WAT?  I don't iike this test.
Why is BluetoothReceive None and not, say, undefined?
This cannot go in without adding a comment there, at least.

In Python, you generally ask for forgiveness rather than permission.
Testing for the availability of something is asking for permission.
I think I'd rather like to see a `try..catch` around Bluetooth portions.